src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String value;
```

Comment:
You would like to use some other variable names. value is too generic. Something like `_block` would be better.